src/i18n/en-us/index.js

```diff
@@ -367,7 +370,7 @@ export default {
             network: 'Network',
             open_language_switcher: 'Open language switcher',
             search_failed: 'Search failed, please enter a valid search term.',
-            search_placeholder: 'Address, Tx, Block',
+            search_placeholder: 'Search anything on Telos',
```

Comment:
You didn't make the same change in the rest of the languages, and I can't create suggestions because is untouched code, so I'm not allowed.